test/public_keys

```diff
@@ -0,0 +1,2 @@
+{pubkey,"112YR4mqBrc6DxrVonKsYE6bb6TGab3K1vVfpF8yAebRSd5YgZDy"}.
```

Comment:
Is this a random public key or did it come from a real hotspot?  This value seems to be here more for unit testing rather than as a default. I am okay with that.

Reply:
random key from a local desktop run.